tests/parametric/test_dynamic_configuration.py

```diff
@@ -56,7 +56,6 @@ def _default_config(service: str, env: str) -> Dict[str, Any]:
             "tracing_debug": None,
             "tracing_service_mapping": None,
             "tracing_sampling_rules": None,
-            "span_sampling_rules": None,
```

Comment:
We just removed this because it was never supposed to be there?

Reply:
yeah. we do not have span rules support.